.nvmrc

```diff
@@ -1 +1 @@
-v8.12.0
\ No newline at end of file
+v8.12.0
```

Comment:
the latest lts is v12.16.0

Reply:
> **Aside** — So I made a mess and accidentally delete this file… then tried to undo that from the history of this PR.

I think we ran into problems with people making top-level changes incidentally, and decided this kind of change should be a separate PR that everyone can pick up (rebase?) in their ongoing PRs.

So imho this nicely aligns with the incremental flows discussion thread on slack… sounds fair?

Reply:
@MylesBorins any insights on why we `nvm` in `v8.12.0`… is it fair to open a separate PR to `v12.16.0`?

Reply:
No idea why, probably an artifact from the original proof of concept. Travis and cloud build are both using Node 10. We should do a separate pr to remove the NVM rc and perhaps update Travis / cloud build to 12

Reply:
Opened #419